src/test/java/com/twilio/twiml/FaxResponseTest.java

```diff
@@ -80,4 +80,43 @@ public void testElementWithTextNode() {
             elem.toXml()
         );
     }
+
+
+    @Test
+    public void testElementWithGenericNode() {
+        GenericNode.Builder genericBuilder = new GenericNode.Builder("genericTag").addText("Some text");
+        GenericNode node = genericBuilder.build();
+
+        MessagingResponse.Builder builder = new MessagingResponse.Builder();
+        MessagingResponse elem = builder.addChild(node).build();
```

Comment:
These should be `FaxResponse` here

Reply:
where did those come from? isn't this auto-generated?

Reply:
Yes and yes?